server/GameOn/GameOn.Common/Models/Player.cs

```diff
@@ -23,12 +23,16 @@ public Player(User user)
             ObjectId = user.ObjectId;
             Surname = user.Surname;
             TenantId = user.TenantId;
+            
+            // Initial Ranking score is 1500 by convention.
+            // TODO: Make the starting Elo score configurable by Tournament
+            RankingScore = 1500;
         }
 
         /// <summary>
-        /// The ranking score for this player as calculated by the ranking algorithm. Null is no rank.
+        /// The ranking score for this player as calculated by the ranking algorithm.
         /// </summary>
-        public int? RankingScore { get; set; }
```

Comment:
I got bitten by this as you and Olaf predicted! Got a null exception because RankingScore was not set. Fix is to set a RankingScore when Player is created. Unfortunately this is a breaking change in prod...